db/migrate/20230621142546_add_columns_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddColumnsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :full_name, :string
+    add_column :users, :avatar_url, :string
+    add_column :users, :provider, :string
```

Comment:
if you are going to scale this to multiple oauth providers, shouldn't this be a separate table along with the uid?

Reply:
agree, that's what I'm trying in the other repo, once we discussed it I will implement it here